walk.go

```diff
@@ -313,6 +320,20 @@ func (s *PromQLSmith) walkLabelReplace(expr *parser.Call) {
 	expr.Args[4] = &parser.StringLiteral{Val: "(.*)"}
 }
 
+func (s *PromQLSmith) walkSortByLabel(expr *parser.Call) {
+	expr.Args = make([]parser.Expr, 0, len(expr.Func.ArgTypes))
+	expr.Args = append(expr.Args, s.Walk(expr.Func.ArgTypes[0]))
+
+	cnt := 0
+	// randomly pick two labels
+	for _, name := range s.labelNames {
+		if cnt < 2 {
+			expr.Args = append(expr.Args, &parser.StringLiteral{Val: name})
+			cnt++
+		}
+	}
```

Comment:
There are 2 issues here:
1. there is no randomization since `s.labelNames` are always the same order
2. For refernece you can check `walkLabelJoin`. We try to get the underlying vector selector and infer what labels it might have and pick labels there.  This increases the chance where the labels used in the query actually makes sense for the series